modules/apps/29-fee/ibc_middleware_test.go

```diff
@@ -766,6 +787,22 @@ func (suite *FeeTestSuite) TestOnTimeoutPacket() {
 					Add(packetFee.Fee.TimeoutFee...)
 			},
 			false,
+			true,
+		},
+		{
+			"fail on no distribution by escrow account out of balance",
```

Comment:
nice test case, maybe we could check that the fee module gets locked and the fees don't get deleted from state. I think we just need to add a boolean variable to this func `expLocked` and then check that the `IBCFeeKeeper.IsLocked` (matches our expected value